app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -3,113 +3,115 @@
   <div class='alert alert-error'><%= flow_session[:error_message] %></div>
 <% end %>
 
-<h1 class='h3 my0'><%= t('doc_auth.headings.welcome') %></h1>
-<p class='mt-tiny mb3'><%= t('doc_auth.info.welcome') %></p>
-<h1 class='h3 mb2'><%= t('doc_auth.instructions.welcome') %></h1>
+<%= render 'shared/maintenance_window_alert' do %>
+  <h1 class='h3 my0'><%= t('doc_auth.headings.welcome') %></h1>
```

Comment:
ditto, this is just indenting & wrapping in a block (recommend that you hide whitespace for this diff)